src/votekit/election_state.py

```diff
@@ -114,15 +108,14 @@ def changed_rankings(self) -> Optional[dict]:
         if not self.previous:
             raise ValueError("This is the first round, cannot compare previous ranking")
 
-        else:
-            prev_ranking = self.previous.get_rankings()
-            curr_ranking = self.get_rankings()
-            if curr_ranking == prev_ranking:
-                print("No changes in ranking")
-
-            changes = {}
-            for index, candidate in enumerate(curr_ranking):
-                if candidate != prev_ranking[index]:
-                    prev_rank = prev_ranking.index(candidate)
-                    changes[candidate] = (prev_rank, index)
-            return changes
+        prev_ranking = self.previous.get_rankings()
+        curr_ranking = self.get_rankings()
+        if curr_ranking == prev_ranking:
+            print("No changes in ranking")
```

Comment:
generally speaking print statements like this don't belong in functions since they'll clutter up downstream output

if this is an error condition, consider raising an exception, or maybe you just want to return [] here to avoid executing the remaining code

Reply:
Ok I'm changing it to return {} for now since we'd otherwise return a dictionary